src/main/java/Class/Task.java

```diff
@@ -0,0 +1,23 @@
+package Class;
+
+public class Task {
```

Comment:
This class looks easy to read and is [simple](https://nus-cs2113-ay2021s2.github.io/website/se-book-adapted/chapters/codeQuality.html#practice-kissing). Good job 👍